guides/source/active_record_querying.md

```diff
@@ -2178,12 +2178,12 @@ You can also use `any?` and `many?` to check for existence on a model or relatio
 
 ```ruby
 # via a model
-Order.any?   # => SELECT 1 AS one FROM orders
-Order.many?  # => SELECT COUNT(*) FROM orders
+Order.any?   # => SELECT 1 AS one FROM orders LIMIT 1
+Order.many?  # => SELECT COUNT(*) FROM (SELECT 1 AS one FROM orders LIMIT 2) subquery_for_count
 
 # via a named scope
-Order.shipped.any?   # => SELECT 1 AS one FROM orders WHERE orders.status = 0
-Order.shipped.many?  # => SELECT COUNT(*) FROM orders WHERE orders.status = 0
```

Comment:
I removed them because I thought it would be cleaner, though I wasn't sure tbh 👍 

I re-added them and squashed commits 👍

Reply:
> I removed them because I thought it would be cleaner, though I wasn't sure tbh

I see what you mean.  The rendered output is getting a little crowded.  What if we omit the SQL aliases, and put the SQL statements on separate lines?  (I've added suggestions to illustrate.)

Reply:
I updated it 🏄 .

You are right, it looks cleaner now without the aliases.

I guess this final version is better in terms of readability, we are on the other hand saying that:
 - it is not necessary for the reader to know the exact query executed from the guides (as we are omitting the aliases)
 - we are saying in a way that it is more useful that the reader knows there is a limit in the query than knowing there are aliases in the real query

Reply:
> * it is not necessary for the reader to know the exact query executed from the guides (as we are omitting the aliases)
> * we are saying in a way that it is more useful that the reader knows there is a limit in the query than knowing there are aliases in the real query

I think that is fine. :+1:  Those aliases don't really provide much information.  But it can be useful to know that Rails is smart enough to include a `LIMIT`, so that we don't have to worry about adding it ourselves.